Tools/TestWebKitAPI/Tests/WebKitGLib/TestWebKitWebView.cpp

```diff
@@ -1411,6 +1412,35 @@ static void testWebViewIsWebProcessResponsive(WebViewTest* test, gconstpointer)
     g_assert_true(webkit_web_view_get_is_web_process_responsive(test->m_webView));
 }
 
+static void testWebViewIsWebProcessUnresponsive(WebViewTest* test, gconstpointer)
+{
+    static const char* hangHTML =
+        "<html>"
+        " <body>"
+        "  <script>"
+        "   setTimeout(function() {"
+        "    var start = new Date().getTime();"
+        "    var end = start;"
+        "     while(end < start + 14000) {"
```

Comment:
Can you can simplify this to `while(true)` since you don't need it to ever become responsive again for this test?

Reply:
hmm, if I use `while(true)`, all tests passed, but then the TestWebKitWebView hangs. I run tests:
`Tools/Scripts/run-gtk-tests --release /app/webkit/WebKitBuild/GTK/Release/bin/TestWebKitAPI/WebKitGTK/TestWebKitWebView`
After last test:
`  /webkit/WebKitWebView/web-extension-mode:                           PASS`
nothing happens - it just hangs.
If I use previous version with `while(end < start + 14000)` all work as expected, after last test I have:
```
  /webkit/WebKitWebView/web-extension-mode:                           PASS
Ran 33 tests of 33 with 33 successful

```
I have to check it.